cylc/flow/pathutil.py

```diff
@@ -43,6 +43,15 @@
 """Matches relative paths that are explicit (starts with ./)"""
 
 
+EXPLICIT_RELATIVE_PATH_REGEX = re.compile(
+    rf'''
+    ^({re.escape(os.curdir)}|{re.escape(os.pardir)})
+    ({re.escape(os.sep)}|$)
+    ''',
+    re.VERBOSE
+)
+
+
```

Comment:
Bad merge? This has become duplicated

Reply:
Huh, a rare example of git not getting that right or flagging a conflict.